packages/ui/test/forum/hooks/useForumCategory.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { renderHook } from '@testing-library/react-hooks'
+import React from 'react'
+
+import { useForumCategory } from '@/forum/hooks/useForumCategory'
+import { seedForumCategory } from '@/mocks/data/seedForum'
+
+import { mockCategories } from '../../_mocks/forum'
+import { MockQueryNodeProviders } from '../../_mocks/providers'
+import { setupMockServer } from '../../_mocks/server'
+
+const renderUseForumCategory = async (id: string) => {
+  const { result, waitForNextUpdate } = renderHook(() => useForumCategory(id), {
+    wrapper: ({ children }) => <MockQueryNodeProviders>{children}</MockQueryNodeProviders>,
+  })
+  while (result.current.isLoading) {
+    await waitForNextUpdate()
+  }
+  return result.current
+}
+
+describe('useForumCategory', () => {
+  const mockServer = setupMockServer({ noCleanupAfterEach: true })
+
+  beforeAll(() => {
+    mockCategories.map((category) => seedForumCategory(category, mockServer.server))
+  })
+
+  it('Basic details', async () => {
+    const { forumCategory } = await renderUseForumCategory('0')
+    expect(forumCategory?.title).toBe('Category 0')
+    expect(forumCategory?.description).toBe('Root category')
+    expect(forumCategory?.id).toBe('0')
+    expect(forumCategory?.parentCategories).toEqual([])
+  })
+
+  describe('Parent categories', () => {
+    it('Query depth exceeds parents depth', async () => {
```

Comment:
```suggestion
    it('One parent', async () => {
```

Reply:
Query depth is an implementation detail IMO, so just call that in a simpler way.